harvard-lts/CURIOSity
# Arabic a11y issues

**Arabic a11y issues**
* * *

**Github Ticket**: https://github.com/harvard-lts/CURIOSity/issues/406

# What does this Pull Request do?
Adds updates to the curiosity site when language is switched to Arabic. For now, these updates are targeting the collection "[Coptica at Harvard](https://curiosity.lib.harvard.edu/coptica-at-harvard)".

# How should this be tested?

A description of what steps someone could take to:
* Spin up Curiosity locally from this branch and log in
* Add “Arabic” as a language to one of your collections (I chose the demo collection) and switch to Arabic to view the following changes:
    * You should notice more padding between the user name and language dropdown menu
    * Page direction has been switched to read right-to-left
        * Everything should look like it’s been flipped from the English version (menu nav on the right, search bar on the left) 
        * Any text that was left-aligned is also now right-aligned
        * If you go to edit text on a page, the cursor should be on the right side
        * Periods (.) at the end of sentences should appear on the right side
    * Letter-spacing
        * Headings: all letter-spacing has been removed so that Arabic words have no space between letters; this also means that any headings left in English will also have no letter-spacing
        * Letter-spacing has also been removed from sidebar navigation, filter rail categories, buttons, and metadata category titles (these appear on both search results and item detail pages)
    * Font-size has been increased for the body font, navigation, and metadata category titles (if you inspect, they should be 20px)
    * Sidebars that have subpages in their nav should appear in italics
    * Updated translations
        * Search (in searchbox) --> بحث
        * Contacts --> للاتصال
        * Results per page -->  نتيجة بكل صفحة
        * Items (on cards) --> نتيجة (in order to see this, you'll have to import the spotlight.ar.yml file that I added; Exhibit dashboard --> Translations; I get an error when I try to get to the dashboard when I'm in Arabic, so I switched back to English in order to upload this)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@VanessaVenti @enriquediaz
